### DISCUSSION TITLE
Velero | Backup  | 2 S3 buckets | Available Options

### DISCUSSION THREAD

**Discussion by sp-today** [vmware-tanzu/velero#7134] [Community support Q&A] [answered]
I want to achieve the Velero backup in separate S3 buckets (one for daily and one for monthly). Is it possible? If so, what are the available options for consumption?

**Comment by sseago** [comment, marked as answer]
You can create multiple BackupStorageLocations. So if you do that, you can configure your dailybackups for one of them and your monthly backups for the other.
